### PR TITLE
Proyect review

### PR DESCRIPTION
**Pull request by joa-byte**
## Review one of your past projects:

This pull request has been created to play the role of the Code Reviewer.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,70 +1,75 @@
...
-    /* font-size: 1.12rem; */
-    margin-bottom: 0.5rem;
+#checkbox,
+#rad {
+  height: 16px !important;
 }
+
+.input-label-pair {
+  /* display: flex; */
+  /* font-size: 1.12rem; */
```
Comment (joa-byte) [inline css/style.css:73]: Please do not leave unused code as a comment

File: css/style.css [hunk 2]
```diff
@@ -1,70 +1,75 @@
...
-    height: 16px !important;
+button {
+  background-color: #00a105;
+  /* Green */
+  border: none;
+  border-radius: 6px;
+  color: white;
+  text-align: center;
+  /* text-decoration: none; */
+  display: inline-block;
```
Comment (joa-byte) [inline css/style.css:62]: Please do not leave unused code as a comment, as it makes the code look "dirty".

File: index.html [hunk 3]
```diff
@@ -20,7 +20,6 @@ <h1>freeCodeCamp Survey Form</h1>
           <form id="survey-form">
             <div id="survey-id">
             <label for="fname">Name:</label><br>
-            
             <input type="text" id="user-name" name="user-name" placeholder="Enter your name"><br>
             <label for="fname">E-mail:</label><br>
             <input type="email" id="email" name="email" placeholder="Enter your email"><br>
```
Comment (joa-byte) [inline index.html:25]: Kindly try to mark the different input sections as required. As is information that shouldn't be skipped.

**Review by joa-byte — COMMENTED** [review]
## Hey! great job so far!
The project looks great, it:
- runs without linter errors
- doesn't have any bugs
- implements the correct flow 


There are a few more changes to add to the protect for this to be ready:

- [ ] Mark the different inputs of the form as required
- [ ] Delete unused code

I'll leave you now so you can continue with your project.
Cheers and Happy coding!👏👏👏

**Review by mugishasam123 — APPROVED** [review]
Hi @joa-byte , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Nice use of inline comments
✔️ Good summary describing required changes


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**